### ISSUE TITLE
error: Invalid option for AuthMethod in config file. Use "basic" or "oauth2"

### ISSUE THREAD

**Issue by atenie** [lfos/calcurse#149]
Hi everyone, I tried following the instructions here to sync with Google Calendar: https://calcurse.org/files/calcurse-caldav.html

```
[General]
Binary = calcurse
Hostname = apidata.googleusercontent.com
Path = /caldav/v2/<...>@gmail.com/events/
AuthMethod = basic
InsecureSSL = No
SyncFilter = cal

DryRun = No
Verbose = Yes

# I tried both with this and without
[Auth]
Username = <...>
Password = <...>

[OAuth2]
ClientID = <...>.apps.googleusercontent.com
ClientSecret = <...>
Scope = https://www.googleapis.com/auth/calendar
RedirectURI = http://127.0.0.1
```

I keep getting an error 401 for me trying to sync with Google Calendar by going the `python3 calcurse-caldav --init keep-remote --authcode <...>'`. I also got a lot longer authcode than in the documentation.
If I go `calcurse-caldav  --init keep-remote --authcode <...>`  I get a `unrecognized arguments: --authcode <...>` error.
Currently running Xubuntu 18.04. Is there any problem here that I'm missing?

**Comment by lfos** [comment]
You have to switch `AuthMethod` from `basic` to `oauth2` if you want to sync with Google calendar (first step in the `OAuth2 Authentication` settings in the calcurse-caldav README).

**Comment by atenie** [comment]
I did just that. It still didn't work.

Another thing I noticed in the past days, is that Google changed their Calendar API. So...maybe a bug from that?

**Comment by lfos** [comment]
"It still didn't work." is not very helpful. Please provide details (error messages, maybe even debug logs) such that we have a chance to figure out what is going on.

**Comment by atenie** [comment]
Ok, so now I hope I can be a bit more precise:
Full config is the one I wrote in the first post with `AuthMethod= oauth2`
In order to get my code from google, I ran `python3 calcurse-caldav.py --init=keep-remote`
I got my code.
Then I ran `python3 calcurse-caldav.py --init=keep-remote --authcode <...>`

The error I got was the following:
```
Connecting to apidata.googleusercontent.com...
error: Invalid option for AuthMethod in config file. Use "basic" or "oauth2"
```
Running with `--debug` didn't go anywhere, sadly. It threw me the exact same error everywhere.
